docs/CONFIG-VARS.md

```diff
@@ -300,6 +302,8 @@ When `storage_type=standard`, a NFS Server VM is created, only when these variab
 | Name | Description | Type | Default | Notes |
 | :--- | ---: | ---: | ---: | ---: |
 | create_nfs_public_ip | Add public ip to the NFS server VM | bool | false | |
+| nfs_public_ip_allocation_method | Defines the allocation method for the public IP address | string | "Static" | Possible values are "Static" or "Dynamic". Only used with `create_nfs_public_ip=true` |
```

Comment:
Looking for a boolean flag here: `create_nfs_static_ip` this matched the other items. What/how do you create a private static ip? Asking as you have public in the name.

Reply:
There is no way to create a private static IP AFAIK, which is why I put public in the name, to show that it was only used with the create_nfs_public_ip flag. But I am happy to follow whatever naming conventions you think are best and show that it only is used in the public IP flow in the documentation.

Reply:
Think we'll be going with `enable_jump_static_ip` and `enable_nfs_static_ip` But that looks good.